assets/queries/cloudFormation/user_data_base64_contains_encoded_private_key/test/positive.yaml

```diff
@@ -0,0 +1,13 @@
+---
+Resources:
+  myLaunchConfig:
```

Comment:
Change "myLaunchConfig" to "myLaunchConfig4"

Reply:
done